tests/test_amp.py

```diff
@@ -9,20 +9,23 @@
     LightningTestModel,
 )
 from pytorch_lightning.utilities.debugging import MisconfigurationException
-from . import testing_utils
+from .utils import (
```

Comment:
it's clear to not import individual functions like this. It should be clear where this function lives and it's in testing_utils, not in this file.

Let's go back to:
```
testing_utils.some_func()
```